addons/eventhandlers/functions/fnc_delayExplosionEH.sqf

```diff
@@ -0,0 +1,39 @@
+#include "script_component.hpp"
+/*
+ * Author: Lambda.Tiger
+ * Delayed reaction to explosions. The delay is calculated on
+ * a) A base line of reaction time
+ * b) Distance from the explosion
+ * c) AI general skill
+ *
+ * Arguments:
+ * BIS Explosion EH
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [BIS EXPLOSION EH ARGS] call lambs_eventhandlers_fnc_delayExplosionEH;
+ *
+ * Public: No
+*/
+params [
+    ["_unit", objNull],
+    "",
+    ["_explosionSource", objNull]
+];
+
+if (!alive _unit ||
```

Comment:
we should move/do more of the checks we do in the [here](https://github.com/nk3nny/LambsDanger/blob/master/addons/eventhandlers/functions/fnc_explosionEH.sqf#L22) before we run the waitAndExecute

Reply:
I agree, but I think a few of these could change in time:
- `!GVAR(ExplosionEventHandlerEnabled)` - should have been there
- `!alive` - covers `isNull` - probably could be replaced with `{!(_unit call EFUNC(main,isAlive))`
- `{!isNul objectParent _unit}` - could be added, but units may be getting in/out and the transition happens in time
- `{(stance _unit) isEqualTo "PRONE"}` - a thing that could be changing in that ~0.15-0.3 ms
- `{_unit getVariable [QGVAR(explosionReactionTime), 0] > time}` -  delay could be computed and then checked against this statement

I'll make the changes I think I made clear I think are good ideas, but for checking if a unit is in a vehicle or is prone I could go either way on.